server2/Arcadia.Assistant/Arcadia.Assistant.Web/Program.cs

```diff
@@ -15,13 +20,17 @@ private static void Main()
         {
             try
             {
-                // The ServiceManifest.XML file defines one or more service type names.
-                // Registering a service maps a service type name to a .NET type.
-                // When Service Fabric creates an instance of this service type,
-                // an instance of the class is created in this host process.
+                var configurationPackage = FabricRuntime.GetActivationContext().GetConfigurationPackageObject("Config");
+                var loggerSettings = new LoggerSettings(configurationPackage.Settings.Sections["Logging"]);
 
                 ServiceRuntime.RegisterServiceAsync("Arcadia.Assistant.WebType",
-                    context => new Web(context)).GetAwaiter().GetResult();
+                    context =>
+                    {
+                        var loggerFactory = new LoggerFactoryBuilder(context).CreateLoggerFactory(loggerSettings.ApplicationInsightsKey);
```

Comment:
This differs from the way all other SF configuration options are used in Web. Just take AppInsights key the same way, from the configuration object

Reply:
Missunderstanding is appear. All configuration load in Startup constructor. But its Program class and Startup object hasnt been yet at this moment. 
At the second, class LoggerSettings used for your 'code formatting' promis, of course this single parameter can be readed without class proxing.